src/components/tipRecords/SoundCloudEmbed.vue

```diff
@@ -49,14 +54,21 @@ export default {
   data() {
     return {
       isPlaying: false,
+      showCookiesDialog: false,
     };
   },
+  computed: {
+    isAllowed() {
+      return this.$store.state.cookiesConsent.SoundCloud;
+    },
+  },
```

Comment:
```suggestion
  computed: mapState({
    isAllowed: 'cookiesConsent.SoundCloud',
  }),
```
won't it work this way?